Cargo.toml

```diff
@@ -7,17 +7,18 @@ build = "build.rs"
 description = "A simple GUI for Neovim."
 
 [features]
-default = ["sdl2"]
+default = ["glfw"]
 embed-fonts = []
 sdl2 = ["skulpin/skulpin_sdl2"]
+glfw = ["skulpin/skulpin_glfw"]
 winit = ["skulpin/skulpin_winit"]
 
 [dependencies]
 euclid = "0.20.7"
 font-kit = "0.6.0"
 skribo = { git = "https://github.com/linebender/skribo" }
 lru = "0.4.3"
-skulpin = { git = "https://github.com/aclysma/skulpin" }
+skulpin = { git = "https://github.com/uhuntu/skulpin", branch = "glfw-windows" }
```

Comment:
Get this validated with upstream first.

Reply:
https://github.com/aclysma/skulpin/pull/72

Reply:
https://github.com/uhuntu/neovide/commit/5a924dcfbc26b1d413ce52b1ce1495032b89b6a4

Reply:
https://github.com/aclysma/skulpin/issues/68#issuecomment-699685177